tutorials/object-storage-sse-c-with-secret-manager/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+meta:
+  title: Using Secret Manager to store encryption key for SSE-C
+  description: Learn how to use Secret Manager to store encryption key for Object Storage and SSE-C.
+content:
+  h1: Using Secret Manager to store encryption key for SSE-C
+  paragraph: Learn how to use Secret Manager to store encryption key for Object Storage and SSE-C.
+tags: object-storage secret-manager encryption
+categories:
+  - object-storage
+  - secret-manager
+  - key-manager
+dates:
+  validation: 2025-10-15
+  posted: 2025-10-15
+---
+import Requirements from '@macros/iam/requirements.mdx'
+
+In this tutorial you will learn how to use Key Manager and Secret Manager to generate and store an encryption key used with [SSE-C](/object-storage/api-cli/enable-sse-c/) to encrypt and decrypt objects stored in a Scaleway Object Storage bucket.
+
+<Requirements />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [Object Storage bucket](/object-storage/how-to/create-a-bucket/)
+- Installed and initialized the [AWS CLI](/object-storage/api-cli/object-storage-aws-cli/)
+
+The goal here is to use Key Manager to generate the encryption key, store the encryption key in Secret Manager, then use it to encrypt Object Storage objects SSE-C.
+
+## Generating the encryption key
+
+Run the following commands to create a key in Key Manager, generate the encryption key, then store it in Secret Manager.
+
+```bash
+KEY_ID=$(scw keymanager key create -o template="{{.ID}}")
```

Comment:
It could be interesting to add one sentence per command to explain what it does: 
- Generate a KMS key - it will be used as a KEK
- Create a DEK derived from the KMS
- Store the DEK in the Secret Manager  (2 last commands)